src/track-gpc-signal.ts

```diff
@@ -0,0 +1,21 @@
+import type { ConsentState } from '@guardian/consent-management-platform/dist/types';
+import { log } from '@guardian/libs';
+import { EventTimer } from './event-timer';
+
+/**
+ * Collect metrics on gpcSignal presence and value
+ * https://globalprivacycontrol.github.io/gpc-spec/
+ */
+const initTrackGpcSignal = (consentState: ConsentState) => {
+	// If undefined we set the property value to -1, false is 0, true is 1
+	const gpcSignal =
+		consentState.gpcSignal === undefined ? -1 : +consentState.gpcSignal;
```

Comment:
Do we need to coerce with `+`?

```suggestion
		consentState.gpcSignal === undefined ? -1 : consentState.gpcSignal;
```

Reply:
Yes, this seems to be required otherwise the build fails complaining i've got the wrong type here.